go.mod

```
@@ -4,5 +4,12 @@ go 1.21
 
 require (
 	github.com/dustin/go-humanize v1.0.1
-	golang.org/x/sys v0.19.0
+	github.com/fatih/color v1.16.0
+	github.com/minio/pkg v1.7.5
```

```suggestion
	github.com/minio/pkg/v3 v1.7.5
```

You would have to use the `pkg/v3` instead to avoid some CVEs